### ISSUE TITLE
Sync Issues — Again

### ISSUE THREAD

**Issue by stefaniwhylie** [nylas/N1#3019]
Looks like I'm having sync issues with N1 again.  This time it's not so much that nothing will sync, but that some changes don't seem to be pushing to Gmail (including Google Apps).  So far, this has affected all of my Gmail and Google Apps accounts.  I've also tried removing and adding the accounts again, but that doesn't resolve the issues.

I noticed it primarily with archiving emails and also with applying labels.  I've also set up new labels in Gmail directly, and those labels aren't showing in N1.  I don't really care so much about the labels (I mean, I do but that's not really urgent), but my main concern is being able to archive emails.  They are removed from my Inbox in N1, but since search is handled on Gmail's end, the emails I archived will still show up when I search.  They also do show up in the inbox if I check Gmail's web client. 

I've been a big, vocal supporter since the beginning and even asked some folks on my team to switch to a pro account, but after having multiple sync issues (most recent was with Gmail syncing and that lasted more than two weeks), I'm starting to feel like I can't fully trust that N1 will do what it says it's doing, which is just horrible considering this is my primary form of communication for work. If I can't rely on it, then it's pretty useless for me right now.  

Here are some screenshots that will hopefully help. 

OSX 10.11.6
<img width="314" alt="screenshot 2016-11-07 18 15 32" src="https://cloud.githubusercontent.com/assets/15056129/20056412/964d9068-a518-11e6-8e90-3fed4b223871.png">
<img width="315" alt="screenshot 2016-11-07 18 16 22" src="https://cloud.githubusercontent.com/assets/15056129/20056413/964e962a-a518-11e6-9e98-8603e0bf05ae.png">

Version 0.4.201-4cdd3ed

**Comment by grinich** [comment]
Any more info you can tell us about the times you have been seeing sync issues?

**Comment by stefaniwhylie** [comment]
Sadly, not much else. I did edit the name of the labels in Gmail's web client and they synced OK after editing.  The only thing I can add is that I've been going through my inbox to clear out some old emails and have been using the "after:2016/MM/DD Label:inbox" to search and have been archiving stuff there.  I have been using this search on the combined inbox (3 separate email accounts), so I'm not sure if the issue is archiving items from search results? 

After re-adding the email accounts, the previously archived emails didn't show up in my inbox in N1, but they were still there when I searched for that date range or checked in Gmail's web client.  

I'll update this ticket if I notice anything else.  Thanks for the response, Michael.
